<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dron</class>
 <widget class="QWidget" name="Dron">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>755</width>
    <height>373</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dron</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>Tab 1</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout">
       <item row="1" column="0">
        <widget class="QGroupBox" name="gb_connection">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="title">
          <string>Connection</string>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="QLabel" name="l_conn">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>Disconnected</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_4">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="pb_conn_2">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="text">
             <string>Connect</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pb_disc_2">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>Disconnect</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QFrame" name="frame_2">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QGroupBox" name="gb_actual_2">
            <property name="minimumSize">
             <size>
              <width>80</width>
              <height>0</height>
             </size>
            </property>
            <property name="title">
             <string>Actual</string>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_8">
             <item>
              <widget class="QLabel" name="l_z_2">
               <property name="text">
                <string>Z: </string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="l_x_2">
               <property name="text">
                <string>X: </string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="l_y_2">
               <property name="text">
                <string>Y: </string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="l_theta_2">
               <property name="text">
                <string>Yaw: </string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QGroupBox" name="groupBox_3">
            <property name="minimumSize">
             <size>
              <width>80</width>
              <height>0</height>
             </size>
            </property>
            <property name="title">
             <string>CF data</string>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_9">
             <item>
              <widget class="QLabel" name="l_vz_2">
               <property name="text">
                <string>Thrust:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="l_vx_2">
               <property name="text">
                <string>Roll:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="l_vy_2">
               <property name="text">
                <string>Pitch:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="l_vtheta_2">
               <property name="text">
                <string>Yaw:</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QGroupBox" name="gb_target_2">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="title">
             <string>Target x,y,z,kat</string>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_10">
             <item>
              <widget class="QLineEdit" name="en_z_2">
               <property name="text">
                <string>0.4</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="en_x_2">
               <property name="text">
                <string>0</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="en_y_2">
               <property name="text">
                <string>0</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="en_theta_2">
               <property name="text">
                <string>0</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QGroupBox" name="groupBox_4">
            <property name="minimumSize">
             <size>
              <width>100</width>
              <height>0</height>
             </size>
            </property>
            <property name="title">
             <string>Errors</string>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_11">
             <item>
              <widget class="QLabel" name="l_thrust_a_2">
               <property name="text">
                <string>Ez:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="l_roll_a_2">
               <property name="text">
                <string>Ex:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="l_pitch_a_2">
               <property name="text">
                <string>Ey:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="l_yaw_a_2">
               <property name="text">
                <string>Eyaw:</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QGroupBox" name="gb_ctrl_data_2">
            <property name="minimumSize">
             <size>
              <width>100</width>
              <height>0</height>
             </size>
            </property>
            <property name="title">
             <string>Control data</string>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_12">
             <item>
              <widget class="QLabel" name="l_thrust_2">
               <property name="text">
                <string>Thrust: </string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="l_roll_2">
               <property name="text">
                <string>Roll:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="l_pitch_2">
               <property name="text">
                <string>Pitch:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="l_yaw_2">
               <property name="text">
                <string>Yaw:</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_3">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QGroupBox" name="gb_control_2">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="title">
             <string>Control</string>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_13">
             <item>
              <widget class="QCheckBox" name="cb_sendCtrl">
               <property name="text">
                <string>send control</string>
               </property>
               <property name="checked">
                <bool>false</bool>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="pb_start_2">
               <property name="enabled">
                <bool>false</bool>
               </property>
               <property name="text">
                <string>Start</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="pb_update_2">
               <property name="enabled">
                <bool>false</bool>
               </property>
               <property name="text">
                <string>Update</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="pb_stop_2">
               <property name="enabled">
                <bool>false</bool>
               </property>
               <property name="text">
                <string>Stop</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="pb_save">
               <property name="text">
                <string>Save</string>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="verticalSpacer">
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>20</width>
                 <height>40</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>Tab 2</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <widget class="QTableWidget" name="table_log">
         <column>
          <property name="text">
           <string>Time</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>X,Y,Z,Yaw -cam</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>X,Y,Z,Yaw -errors</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>X,Y,Z,Yaw - V</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>TRPY-control</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>TRPY - dron</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>motors</string>
          </property>
         </column>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
